### PR TITLE
Documentation improvement

### PR DESCRIPTION
**Pull request by jihoonson**
Hi,
I've improved some documents by fixing errata and adding some expressions.

[Review on Reviewable](https://reviewable.io/reviews/hyunsik/tokamak/1)

### CODE REVIEW

**Comment by hyunsik** [comment]
I just felt that I should keep the line width even in documentation. Whole lines are marked as modified even if some words are changed.

**Comment by jihoonson** [comment]
Right. Do you want to fix the line width problem first?
If so, I'll update my PR later.

**Comment by hyunsik** [comment]
Sure, ok, I'll first do that.

**Comment by hyunsik** [comment]
Also, I wrote more sections before this PR. I'll commit them too. Could you review the sections too with your PR?

**Comment by hyunsik** [comment]
I've updated the documentation. See the commit log (455fb). Also, I broke the lines with 80 width of each line.

**Comment by jihoonson** [comment]
Sure. Thanks for quick update.
I'll update my patch soon.

**Comment by jihoonson** [comment]
Thanks. I've updated my patch.

File: INTERNAL.md [hunk 1]
```diff
@@ -14,55 +14,63 @@ All terminologies are the unique names for clear understanding.
...
 # JIT Compilation
-(TODO)
-In Tokamak, JitManager generates some JIT code fragments in runtime for 
+In Tokamak, JitManager generates some JIT code fragments in runtime for
 arithmetic, comparison, other logical expressions and predicates.
 
 Access to nested fields
 
-Hash function
+* Various invocation methods
```
Comment (hyunsik) [inline INTERNAL.md]: It moved and is separated into multiple parts. See the following lines.

https://github.com/hyunsik/tokamak/pull/1/files#diff-0fa472b032003b2960b08cb7545de6b0R114

https://github.com/hyunsik/tokamak/pull/1/files#diff-0fa472b032003b2960b08cb7545de6b0R120

**Comment by hyunsik** [comment]
I leaved a trivial comment. Others look good to me.

I also found that the different style of trailing space makes some changes. We need to decide some coding style too :) I know some automatic tools for it. I'll share it later in another ticket.

**Comment by jihoonson** [comment]
Thanks! I've removed that section.
Also, I agree on the need for coding style. Let's do that.

**Comment by hyunsik** [comment]
I just merged the PR. Thank you!

**Comment by jihoonson** [comment]
Thanks!

**Pull request merged**